src/parser/CodeBlock.cpp

```diff
@@ -485,7 +458,9 @@ void InterpretedCodeBlock::computeBlockVariables(LexicalBlockIndex currentBlockI
     }
 
     bi->m_shouldAllocateEnvironment = isThereHeapVariable;
-
+    if (isThereHeapVariable) {
```

Comment:
This if-statement is debug only code. What about wrapping with debug macros?

Reply:
on release mode, compiler will remove unused block I think
but I can use ifdef you have mind

Reply:
I added it